### DISCUSSION TITLE
Import STL--Export boundaryVTK--No motiondata

### DISCUSSION THREAD

**Discussion by hongshanma** [DualSPHysics/DualSPHysics#273] [DUALSPHYSICS_V5.2]
Hi，everybody！
I import STL and apply it in motion. Then i use boundaryVTK to export .vtk file,but it don't save the  motiondata.
I also tried to export .stl file,  neither have motiondata.Exported files are no moving,just stay at one place.
That's my bat code:
set dirout2=%dirout%\boundary%boundaryvtk% -loadstl:11 external_GEO_065b521f_cf18_4b20_9415_d841c354aa42.stl -motiondata %diroutdata% -savevtkdata %dirout2%/Box.vtk -onlymk:11 -savevtkdata %dirout2%/Hull.vtk -onlytype:-all,moving
if not "%ERRORLEVEL%" == "0" goto fail

**Comment by alexbexe** [comment]
You should try to create Case_Actual.vtk with the geometries you created in the XML
You can follow explanation in DualSPHysics_v5.2\doc\guides\XML_GUIDE_v5.2.pdf page 25

Then you can use -loadvtk AutoActual in the command line of BoundaryVTK.exe

**Reply by hongshanma** [reply]
thanks，i sloved this problem.
